src/content/docs/docs.mdx

```diff
@@ -4,38 +4,57 @@ title: Intro to Web Monetization
-:::
+* Obtaining authorization to initiate the payment.
+* Gathering details and issuing instructions to start the money movement process.
+* Creating a payment session.
+* Communicating payment events to the site so the site can optionally respond. For example, the site could [hide ads](/docs/guides/hide-ads) or provide [access to exclusive content](/docs/guides/provide-exclusive-content) for paying visitors. 
+
+### High-level flow
+
+The following image shows the Web Monetization flow at a high level. Some steps have been combined or excluded. A more in-depth explanation is provided in the [Web Monetization flow](/docs/intro/web-monetization-flow) page.
+
+<LargeImg src="/img/diagram-wm-overview.png" alt="High-level overview of Web Monetization flow" />
+
+1. Bob visits a [web monetized page](/docs/guides/monetize-page). The monetization `<link>` element is how the website signals its acceptance of payments to the browser. 
+2. Bob’s browser, either natively or via a browser extension, parses the `<link>` element to get the <Tooltip content='A URL that identifies an Open Payments account' client:load><a href="./resources/glossary#payment-pointer">payment pointer</a></Tooltip> for Alice, the site owner.
+3. Bob’s browser/extension sends requests to Alice’s payment pointer to obtain authorization and instructions for sending a payment.
+4. With authorization granted and payment instructions received, the browser/extension sends requests to Bob’s payment pointer to initiate the outgoing payment.
+5. The Web Monetization flow ends. Payment processing, currency exchange, and settlement occurs between the two accounts via a common payment rail. 
+
+### Specifying payment amounts and currencies
+
+Web Monetization doesn’t allow a website to specify a payment amount or currency. It only allows the site to tell the browser it can accept payments. 
+
+With the help of a <Tooltip content='The entity sending a payment' client:load><a href="./resources/glossary#web-monetization-provider">WM provider</a></Tooltip>, your visitor decides whether to make a payment, how much and how often to pay, and in which currency. Your <Tooltip content='The entity receiving a payment' client:load><a href="./resources/glossary#web-monetization-receiver">WM receiver</a></Tooltip> can then exchange the currency of incoming payments based on what you want to receive. This flexibility allows you and your visitors to choose the monetization methods that best suit your needs.
+
+### Processing and settling payments
+
+Web Monetization’s role is to help coordinate payments. It does not process or settle payments. 
+
+At each end of Web Monetization is an account that supports <LinkOut href="https://openpayments.guide">Open Payments</LinkOut>. The WM provider supplies your visitor with a funded sending account. In some cases, the visitor could even act as their own WM provider. The WM receiver supplies you with a receiving account. 
```

Comment:
> In some cases, the visitor could even act as their own WM provider.

I think we should elaborate on this.

Reply:
The only reason I disagree (for now) is because the docs don't cover how to become a WM provider. The Sending Payments page states, "This page describes the roles of WM providers and WM agents within the Web Monetization ecosystem. It does not explain how to become a provider or develop an agent." The Receiving Payments page says something similar about WM receivers. 

I think the plan is to eventually document how do become a provider/receiver and develop an extension but we aren't there yet. Maybe I should take out the line you quoted?